olp-cpp-sdk-core/include/olp/core/client/OlpClientFactory.h

```diff
@@ -32,21 +32,22 @@ class OlpClient;
 }
 
 /**
- * @brief client namespace
+ * @brief The `client` namespace.
  */
 namespace client {
 /**
- * @brief Factory that creates OlpClient instances that is used for all REST
+ * @brief Creates the `OlpClient` instances that are used for all the REST
```

Comment:
```suggestion
 * @brief Creates the `OlpClient` instances that used for every HTTP request.
```
Not related to the change, as the previous version of the docs was not entirely correct

Reply:
Changed